x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java

```diff
@@ -492,9 +519,21 @@ private void createNewSingletonPolicy(String phaseName, LifecycleAction action,
         client().performRequest(request);
     }
 
+    private void createIndexWithSettingsNoAlias(String index, Settings.Builder settings) throws IOException {
+        // create the test-index index
```

Comment:
I suppose this is a but with the existing code that was most likely copy-paste, but it makes it sound like `test-index` is the index being created